src/cc_helpers.cc

```diff
@@ -110,29 +116,6 @@ certifier::framework::cc_trust_data::cc_trust_data(const string& enclave_type, c
   }
 }
 
-certifier::framework::cc_trust_data::cc_trust_data() {
-  cc_basic_data_initialized_ = false;
-  cc_policy_info_initialized_= false;
-  cc_policy_store_initialized_ = false;
-  cc_service_key_initialized_ = false;
-  cc_service_cert_initialized_ = false;
-  cc_service_platform_rule_initialized_ = false;
-  cc_sealing_key_initialized_ = false;
-  cc_provider_provisioned_ = false;
-  x509_policy_cert_ = nullptr;
-  cc_is_certified_ = false;
-
-  for (int i = 0; i < num_certified_domains_; i++) {
```

Comment:
In the old code, this variable `num_certified_domains_` is used uninitialized.
That's causing a seg-fault.

Reply:
great!